src/signals/incident/components/form/MapSelectors/Asset/__tests__/AssetSelect.test.tsx

```diff
@@ -89,4 +125,74 @@ describe('AssetSelect', () => {
     expect(screen.queryByTestId('assetSelectSelector')).not.toBeInTheDocument()
     expect(screen.queryByTestId('assetSelectSummary')).toBeInTheDocument()
   })
+
+  it('handles setLocation for a successful reverse geocode request', async () => {
+    const geocodedResponse = {
+      id: 'foo',
+      value: 'bar',
+      data: {
+        location: {
+          lat: 52.37377195,
+          lng: 4.87745608,
+        },
+        address: {
+          openbare_ruimte: '',
+          huisnummer: '189A-2',
+          postcode: '1016KP',
+          woonplaats: 'Amsterdam',
+        },
+      },
+    }
+    mocked(reverseGeocoderService).mockImplementation(() =>
+      Promise.resolve(geocodedResponse)
+    )
+
+    render(withAssetSelectContext(<AssetSelect {...props} />))
+
+    userEvent.click(screen.getByText(/kies op kaart/i))
+
+    const assetSelectSelector = screen.getByTestId('assetSelectSelector')
+
+    expect(updateIncident).not.toHaveBeenCalled()
+
+    userEvent.click(assetSelectSelector)
+
+    expect(updateIncident).toHaveBeenCalledTimes(1)
+    expect(updateIncident).toHaveBeenCalledWith({ Zork: [] })
+
+    await screen.findByTestId('assetSelectSelector')
+
+    expect(updateIncident).toHaveBeenCalledTimes(2)
+    expect(updateIncident).toHaveBeenLastCalledWith({
+      location: expect.objectContaining({
+        geometrie: expect.objectContaining({ type: 'Point' }),
+        address: geocodedResponse.data.address,
+      }),
+    })
+  })
+
+  it('handles setLocation for a unsuccessful reverse geocode request', async () => {
+    const geocodedResponse = undefined
```

Comment:
when does this happen? what does it mean?

Reply:
It happens when the reverse geocoder cannot find an address from a GPS coordinate